Add UW venue info

kdillmcfarland opened the pull request.

kdillmcfarland committed:
archive Fred Hutch venue info

kdillmcfarland committed:
Add UW venue info

kdillmcfarland committed:
Create 2024_cfp_banner.png

kdillmcfarland committed:
Update venue.md

kdillmcfarland committed:
Update venue.md

kdillmcfarland committed:
typo

kdillmcfarland committed:
Update faq.md

jadeynryan committed:
Update venue.md

Few minor fixes.

jadeynryan committed:
Update faq.md

jadeynryan reviewed (approved):
Committed a few minor changes:
-  I prefer being explicit (University of Washington instead of U of Washington)
-  couple typos
-  fixed UW google map link.

jadeynryan reviewed (changes requested):
Just realized the Venue button in the navbar still goes to the more info coming soon venue page (https://deploy-preview-149--brave-albattani-75ca02.netlify.app/coming_soon/venue/)

jadeynryan committed:
Update config.toml

Change the venue navbar to the actual venue page instead of the coming soon venue page.

kdillmcfarland commented:
> Just realized the Venue button in the navbar still goes to the more info coming soon venue page (https://deploy-preview-149--brave-albattani-75ca02.netlify.app/coming_soon/venue/)

Yes, I had not turned this on yet because I don't want it live to the website until we have a tickets page as well. This is setup with `UPDATE.sh`

kdillmcfarland committed:
Revert "Update config.toml"

This reverts commit dfbc6a3735108f023cd1335ede463c2447e7cf2d.
